pkg/security/ebpf/c/chmod.h

```diff
@@ -0,0 +1,97 @@
+#ifndef _CHMOD_H_
```

Comment:
Should probably have comment headers on these files as well?

Reply:
Obviously, we can't have the usual headers on top of these :-) So I suggest we just leave it as it